src/app/component/auth/sign-in-new/sign-in-new.component.ts

```diff
@@ -0,0 +1,46 @@
+import { Component, OnInit } from '@angular/core';
+import { AuthService, GoogleLoginProvider } from 'angularx-social-login';
+import { GoogleSignInService } from '../../../service/auth/google-sign-in.service';
+import {UserSuccessSignIn} from '../../../model/user-success-sign-in';
+import {HttpErrorResponse} from '@angular/common/http';
+import {UserOwnSignInService} from '../../../service/auth/user-own-sign-in.service';
```

Comment:
General: please add 2 spaces between brackets in imports

Reply:
Fixed